.github/workflows/test-suite.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: 🏥 Check docs generation
         run: |
-          nix develop --show-trace --command bash -c "npm run docs"
+          nix develop --show-trace --command bash -c "npm --prefix ./theme/ run build && npm run docs"
```

Comment:
Can we have this theme folder inside of `doc/theme`? Also, notice there is an image folder there. If we could reutilize those assets it would be awesome

Reply:
Done